metabase/metabase
# test: do not allow cypress to raise an error on timeline hover

### Description

Attempt to address the issue with cypress left bar which raises errors on hover.

### How to verify

- run any e2e test
- try to hover over different steps
- make sure there is no 100500 errors in the console

### Before

https://github.com/user-attachments/assets/6ecbafed-b2e5-478f-a7a5-64018965708a


### After

https://github.com/user-attachments/assets/972536af-fcc0-4a6f-a7ca-6526c61fcbed
